### PR TITLE
ref: fix fastrcnn test

### PR DESCRIPTION
**Pull request by williamFalcon**

### CODE REVIEW

**Comment by williamFalcon** [comment]
@teddykoker @Borda is there a torchvision issue or something here?

**Comment by teddykoker** [comment]
Hmm this is strange; just tested locally and it is working with torch==1.6.0 torchvision==0.7.0. I'll look into it some more

**Comment by teddykoker** [comment]
```
E           RuntimeError: 
E           object has no attribute nms:
E             File "/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/torchvision/ops/boxes.py", line 40
E                   by NMS, sorted in decreasing order of scores
E               """
E               return torch.ops.torchvision.nms(boxes, scores, iou_threshold)
E                      ~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
E           'nms' is being compiled since it was called from 'batched_nms'
E             File "/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/torchvision/ops/boxes.py", line 82
E                   offsets = idxs.to(boxes) * (max_coordinate + torch.tensor(1).to(boxes))
E                   boxes_for_nms = boxes + offsets[:, None]
E                   keep = nms(boxes_for_nms, scores, iou_threshold)
E                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
E                   return keep
```
This is the error on CI, can't seem to reproduce locally or in Docker

**Comment by Borda** [comment]
> This is the error on CI, can't seem to reproduce locally or in Docker. Seems to be a similar issue [here](https://github.com/pytorch/pytorch/issues/39444), but I'm not sure how to fix exactly as I can't reproduce the error. Any ideas @Borda?

seem to work for me locally too, co I have dropped the cache to be sure, there is no CI cashing issue...

**Comment by Borda** [comment]
seems to be related to https://github.com/pytorch/vision/issues/1916

File: tests/models/test_detection.py [hunk 1]
```diff
@@ -11,17 +11,16 @@ def _collate_fn(batch):
     return tuple(zip(*batch))
 
 
-@pytest.mark.skip(reason='failing test')  # TODO
 def test_fasterrcnn(tmpdir):
-
+    # TODO: fix test
```
Comment (Borda) [inline tests/models/test_detection.py:15]:
```suggestion
```

File: tests/models/test_detection.py [hunk 2]
```diff
@@ -11,17 +11,16 @@ def _collate_fn(batch):
...
     model = FasterRCNN()
 
     image = torch.rand(1, 3, 400, 400)
     model(image)
 
 
-@pytest.mark.skip(reason='failing test')  # TODO
 def test_fasterrcnn_train(tmpdir):
+    # TODO: fix test
```
Comment (Borda) [inline tests/models/test_detection.py:24]:
```suggestion
```

**Review by Borda — APPROVED** [review]

**Pull request merged**